Dockerfile

```diff
@@ -68,38 +76,38 @@ RUN set -x \
     && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
     && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
     && rm -r "$GNUPGHOME" "node-v$NODE_VERSION-linux-x64.tar.gz" SHASUMS256.txt.asc \
-    && apt-get purge -y --auto-remove wget
-
-RUN npm install -g yarn
+    && apt-get purge -y --auto-remove wget \
+    && npm install -g yarn@$YARN_VERSION \
+    && npm cache clear --force
 
-COPY yarn.lock /usr/src/app/
-COPY package.json /usr/src/app/
-RUN yarn install
+COPY package.json yarn.lock /usr/src/zeus/
+RUN yarn install --production --pure-lockfile --ignore-optional \
```

Comment:
Not sure exactly what these mean, but this was for generating a more production build.

Then again, follow up by clearing the cache since it's not needed and wastes space.